composer/core/algorithm.py

```diff
@@ -66,11 +65,9 @@ def backwards_create_graph(self) -> bool:
 
     @abstractmethod
     def match(self, event: Event, state: State) -> bool:
-        """Determines whether this algorithm should run given the current :class:`~.event.Event` and
-        :class:`~.state.State`.
+        """Whether this algorithm should run given the current :class:`~.event.Event` and :class:`~.state.State`.
```

Comment:
```suggestion
        """Determines whether this algorithm should run given the current :class:`~.event.Event` and :class:`~.state.State`.
```

I think removing the verb from these sentences makes them somewhat less clear.